### PR TITLE
Clarify methods for reviewer r1

### PR DESCRIPTION
**Pull request by sdtaylor**
I went thru and picked off the easiest ones. each commit addresses a single comment, or 2 related comments.

### CODE REVIEW

File: manuscript/reviewer_reply.Rmd [hunk 1]
```diff
@@ -31,15 +31,19 @@ Line 41: I think this is generally more common in terrestrial ecology – in fis
 
 Line 62: I really like the recommendations / clarity in Box 1, and think these will be accessible to most ecologists
 
-Line 71-73: I’m re-reading this after commenting on data, and am now a little confused. The bird responses are annual snapshots during the breeding season? And I thought the environmental data you used was annual or quarterly, not monthly?
-https://www.mbr-pwrc.usgs.gov/bbs/grass/genintro.htm
+"Line 71-73: I’m re-reading this after commenting on data, and am now a little confused. The bird responses are annual snapshots during the breeding season? And I thought the environmental data you used was annual or quarterly, not monthly?
+https://www.mbr-pwrc.usgs.gov/bbs/grass/genintro.htm"
+
+--BBS data are generally viewed as annual data, even though the observations happen on a single day. The presence of an individual species is primarily a function of suitable habitat and, for many species, timing of migration. Environmental variables with a shorter temporal scale could likely improve models, such as incorporating cool weather days into the observation model, and would be worth pursuing in a different study.
```
Comment (davharris) [inline manuscript/reviewer_reply.Rmd]: I think we can be clearer here.

File: manuscript/draft.Rmd [hunk 2]
```diff
@@ -144,11 +144,14 @@ correcting for observer effects, as described below.
 
 **Richness data.** Bird species richness was obtained from the North American
 Breeding Bird Survey (BBS) [@pardieck2017] using the Data Retriever Python
-package [@morris2013] and rdataretriever R package [@rdataretriever]. The BBS
-data was filtered to exclude all nocturnal, cepuscular, and aquatic species
-[since these species are not well sampled by BBS methods; @hurlbert2005], as
-well as unidentified species, and hybrids. All data from surveys that did not
-meet BBS quality criteria were also excluded.
+package [@morris2013] and rdataretriever R package [@rdataretriever]. BBS
+observations are three minute point counts made at 50 fixed locations along a
```
Comment (davharris) [inline manuscript/draft.Rmd]: three-minute should be hyphenated because it modifies "point counts"

File: manuscript/draft.Rmd [hunk 3]
```diff
@@ -144,11 +144,14 @@ correcting for observer effects, as described below.
...
+40km route. Here we denote each route as a site and summarize richness as the
```
Comment (davharris) [inline manuscript/draft.Rmd]: This is likely to be confusing to other readers too.  Should we just switch to "route" throughout?

File: manuscript/reviewer_reply.Rmd [hunk 4]
```diff
@@ -72,7 +74,9 @@ Table 1: Just my personal preference, but I’d include the check marks and repl
 
 Line 194-221: It seems for each approach you only considered the full models with all environmental covariates? I’m curious whether any of the covariates appeared to not improve explanatory power, and whether you considered models that dropped these uninformative predictors?
 
-Line 216: Cite randomForest / caret/ party R package – or whatever else you used?
+"Line 216: Cite randomForest / caret/ party R package – or whatever else you used?"
+
+--We've cited this here.
```
Comment (davharris) [inline manuscript/reviewer_reply.Rmd]: Maybe mention that we did cite it elsewhere so we don't look *totally* sloppy.

File: manuscript/draft.Rmd (hunk 3 above)
Comment (sdtaylor) [reply manuscript/draft.Rmd]: maybe "Each site consists of a 50 point counts along a 40km route" ?

**Comment by sdtaylor** [comment]
I'll go through and switch all reviewer comments to quotes and ours to bold after this PR

File: manuscript/reviewer_reply.Rmd (hunk 1 above)
Comment (davharris) [reply manuscript/reviewer_reply.Rmd]: @ethanwhite Any thoughts on how to make this part clearer (both in the manuscript and the response)? It's kind of subtle.

Comment (davharris) [reply manuscript/reviewer_reply.Rmd]: @ethanwhite Actually, in the interest of avoiding merge conflicts, I'll just merge now and create a separate issue.

**Pull request merged**